tsflex/features/segmenter/strided_rolling.py

```diff
@@ -129,23 +129,23 @@ class StridedRolling(ABC):
     )
 
     # Create the named tuple
-    _NumpySeriesContainer = namedtuple(
+    _NumpySeriesContainer = namedtuple(  # type: ignore[name-match]
         "SeriesContainer", ["name", "values", "start_indexes", "end_indexes"]
     )
 
     def __init__(
         self,
         data: Union[pd.Series, pd.DataFrame, List[Union[pd.Series, pd.DataFrame]]],
-        window: T,
+        window: Optional[T],
         strides: Optional[Union[T, List[T]]] = None,
         segment_start_idxs: Optional[np.ndarray] = None,
         segment_end_idxs: Optional[np.ndarray] = None,
         start_idx: Optional[T] = None,
         end_idx: Optional[T] = None,
-        func_data_type: Optional[Union[np.array, pd.Series]] = np.array,
-        window_idx: Optional[str] = "end",
+        func_data_type: Union[np.ndarray, pd.Series] = np.ndarray,
```

Comment:
Wouldn't it be more convenient to make this a literal?

Reply:
Yes!

Reply:
But this would require an API change - I'll submit an issue for this

Reply:
https://github.com/predict-idlab/tsflex/issues/121